### PR TITLE
fix: refactor accessToken process & fix cost-report bug

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]

File: apps/web/src/router/constant.ts [hunk 1]
```diff
@@ -26,3 +22,13 @@ export const ROUTE_SCOPE = {
...
+
+export const EXTERNAL_PAGE_ROUTE = {
+    COST_REPORT_DETAIL: {
+        _NAME: 'cost-report-detail',
+    },
+} as const;
+
+export const EXTERNAL_PAGES = [
+    EXTERNAL_PAGE_ROUTE.COST_REPORT_DETAIL._NAME,
+];
```
Comment (piggggggggy) [inline apps/web/src/router/constant.ts:34]: 👍👍

File: apps/web/src/router/helpers/route-helper.ts [hunk 2]
```diff
@@ -73,9 +73,10 @@ export const processTokenVerification = (to: Route, next: NavigationGuardNext, r
 
     const ROOT_REDIRECT_SKIP_ROUTE_NAMES = [
         AUTH_ROUTE.SIGN_OUT._NAME,
-        AUTH_ROUTE.PASSWORD.STATUS.RESET._NAME,
         ERROR_ROUTE._NAME,
-        ROOT_ROUTE.COST_REPORT._NAME,
+        AUTH_ROUTE.PASSWORD.STATUS.RESET._NAME,
+        ERROR_ROUTE.EXPIRED_LINK._NAME,
+        ...EXTERNAL_PAGES,
```
Comment (piggggggggy) [inline apps/web/src/router/helpers/route-helper.ts:79]: 😇

File: apps/web/src/router/index.ts [hunk 3]
```diff
@@ -35,9 +35,9 @@ const grantAndLoadByCurrentScope = async (scope: GrantScope, workspaceId?: strin
     };
 
     await SpaceRouter.router.app?.$store.dispatch('user/grantRoleAndLoadReferenceData', grantRequest);
-    const grantAcessFailStatus = SpaceRouter.router.app?.$store.state.error.grantAccessFailStatus;
+    const grantAccessFailStatus = SpaceRouter.router.app?.$store.state.error.grantAccessFailStatus;
     return {
-        failStatus: !!grantAcessFailStatus,
+        failStatus: !!grantAccessFailStatus,
```
Comment (piggggggggy) [inline apps/web/src/router/index.ts:40]: 🫢

**Review by piggggggggy — APPROVED** [review]
LGTM!!!!

**Pull request merged**